src/site-layout/SiteLayout.js

```diff
@@ -51,6 +55,8 @@ const FooterWrapper = styled.div`
 `;
 
 function SiteLayout({ tenantId }) {
+  const showNav = useBreakpoint(false, ["tablet+", true]);
```

Comment:
It looks like we start showing the Navbar again when the screen size reaches a width of 640px.  I'm not sure if this value was specified somewhere, but I personally think we should increase it a bit if we can.  When I view the app on my phone at 667px (i.e. iPhone 7 turned horizontally) it feels like everything gets smooshed to the right as I scroll down because the navigation is still present:

<img width="641" alt="Screen Shot 2020-07-16 at 4 33 31 PM" src="https://user-images.githubusercontent.com/25503/87720456-f8ad3c00-c782-11ea-9a8f-399f7a605b65.png">

Reply:
oh yeah we can definitely override the default breakpoints. nothing was specified in the designs so I just left the defaults in place; i may not have hit this exact combination of resolution and orientation in my ad hoc testing